components/atoms/CopyToClipboard/CopyToClipboardStyles.ts

```diff
@@ -0,0 +1,26 @@
+import ContentCopyIcon from 'mdi-react/ContentCopyIcon'
+import styled from 'styled-components'
+
+export const StyledCopyMessage = styled.span`
+    color: #a112ff;
```

Comment:
What do you think about making this `#3826cc` too?

Reply:
Oh yes, this should definitely be the same color — thanks for catching!